autogpt/agent/agent.py

```diff
@@ -87,6 +87,7 @@ def start_interaction_loop(self):
         command_name = None
         arguments = None
         user_input = ""
+        auto_confirm = False
```

Comment:
@AbTrax I am on your branch, running python -m autogpt --debug and I keep typing "s"
And it keeps sending back a very long sentence if it agrees with the next action.
And on top of that it doesn't do the actual action, this means if I keep typing s it will keep doing nothing

Here is what we want: 
typing s should do the next action if supervisor agrees, otherwise it should give clear and concise sentence to nudge the agent in the right direction.

Please look at my debug files to see the issue for yourself.
[DEBUG.zip](https://github.com/Significant-Gravitas/Auto-GPT/files/11421250/DEBUG.zip)

Also you didn't pick the work of @Torantulino in his commit, this commit looks pretty good